commands/team.go

```diff
@@ -353,10 +355,13 @@ func modifyTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 }
 
 func restoreTeamsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	var errs *multierror.Error
 	teams := getTeamsFromTeamArgs(c, args)
 	for i, team := range teams {
 		if team == nil {
-			printer.PrintError("Unable to find team '" + args[i] + "'")
+			teamErr := fmt.Errorf("Unable to find team '" + args[i] + "'")
```

Comment:
You can use string interpolation here
```suggestion
			teamErr := fmt.Errorf("Unable to find team %q", args[i]")
```
using `%q` will automatically add `"` around the string.